web/src/routes/TimelineView.svelte

```diff
@@ -33,16 +36,43 @@
 			}
 		});
 	});
+
+	const viewDetail = (
+		clickEvent: MouseEvent & {
+			currentTarget: EventTarget & HTMLLIElement;
+		},
+		eventId: string
+	) => {
+		const target = clickEvent.target as HTMLElement;
+
+		let parent = target.parentElement;
+		if (parent) {
+			while (parent && !parent.classList.contains('timeline')) {
+				const tagName = parent.tagName.toLocaleLowerCase();
+				if (tagName === 'a' || tagName === 'button') {
+					return;
+				}
+				if (tagName === 'p' && String(document.getSelection()).length) {
+					return;
+				}
+				parent = parent.parentElement;
+			}
+		}
+		const noteId = nip19.noteEncode(eventId);
+		window.location.href = `/${noteId}`;
+		return;
```

Comment:
ログインを維持したまま遷移させるのは [`goto`](https://kit.svelte.jp/docs/modules#$app-navigation-goto) で。
```suggestion
		await goto(`/${noteId}`);
```